package/MDAnalysis/core/topologyattrs.py

```diff
@@ -1799,6 +1800,137 @@ def n_fragments(self):
         ('n_fragments', property(n_fragments, None, None,
                                  n_fragments.__doc__)))
 
+    def unwrap(self, box=None, compound='fragments', reference='com',
+               inplace=True):
+        """Move atoms in this group so that bonds within the
+        group's compounds aren't split accross periodic boundaries.
+
+        Parameters
+        ----------
+        box : array_like or None, optional
```

Comment:
maybe just force this to always be `self.dimensions`?  Thinking of how to reduce the complexity of this

Reply:
`*Group.wrap()` has the `box` kwarg as well, so I wanted to stay consistent there...

Reply:
Ok what I said doesn't make sense at all since `box` is never really used except for the optimization check, which is potentially inconsistent with what `make_whole()` does and will go there anyway. I'll get rid of the `box` kwarg.

Reply:
I just couldn't think of a time when you would want to specify a box but not set it as `u.dimensions`